dot/digest/digest.go

```diff
@@ -401,7 +401,7 @@ func newGrandpaChange(raw []types.GrandpaAuthoritiesRaw, delay uint32, currBlock
 	}, nil
 }
 
-func (*Handler) handleBABEOnDisabled(_ *types.ConsensusDigest, _ *types.Header) error {
+func (h *Handler) handleBABEOnDisabled(d types.BABEOnDisabled, _ *types.Header) error {
```

Comment:
what does this function do? looks like it just logs some struct and returns nil.

Reply:
I'm not sure why this is here...